src/Raven.Studio/typescript/viewmodels/database/documents/editDocumentConnectedDocuments.ts

```diff
@@ -52,6 +43,46 @@ class connectedDocuments {
         }
     });
 
+    static connectedDocsTabs = {
+        related: "related",
+        collection: "collection",
+        recent: "recent",
+        starred: "starred"
+    };
+    static currentTab = ko.observable<string>(connectedDocuments.connectedDocsTabs.related);
+
+    constructor(document: KnockoutObservable<document>, db: KnockoutObservable<database>, loadDocument: (docId: string) => void) {
+        this.document = document;
+        this.db = db;
+        this.document.subscribe((doc) => this.onDocumentLoaded(doc));
+        this.loadDocumentAction = loadDocument;
+        virtualGrid.install();
```

Comment:
That, or we can install it once (maybe in shell). It only needs to be installed once. Install will do nothing if it's already installed.